pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -493,9 +493,9 @@ func (c *Controller) matchIGMPType(r *rule, igmpType uint8, groupAddress string)
 	return false
 }
 
-// Validate checks if there is rule to drop or allow IGMP report from a Pod to a group Address, and returns multicast
```

Comment:
rule -> a rule

Reply:
and returns the rule information if exits

Reply:
done